fmdtools/analyze/graph.py

```diff
@@ -681,6 +785,8 @@ def draw_pyvis(self, filename="graph", width=1000, filt=True, physics=False, not
             Whether to display sliders. The default is True.
         physics : Bool, optional
             Whether to use physics during node placement. The default is False.
+        notebook : Bool, optional
+            todo : requires documentation update.
```

Comment:
passes notebook arg to pyviz Network (for displaying in jupyter)

Reply:
Done